Documentation/Metrics/rocksdb_cache_hit_rate_lifetime.yaml

```
@@ -1,7 +1,7 @@
 name: rocksdb_cache_hit_rate_lifetime
 introducedIn: "3.6"
 help: |
-  rocksdb_cache_hit_rate_lifetime
+  Lifetime hit rate of the ArangoDB cache in front of RocksDB
 unit: number
```

Should this be a "ratio"?
```suggestion
unit: ratio
```